integ/BoundaryRectangular/render.ts

```diff
@@ -57,6 +58,7 @@ export function renderDiagramMaker() {
         }
       },
       actionInterceptor: (action: Action, next: Dispatch<Action>, getState: () => DiagramMakerData<{}, {}>) => {
+        console.log(getState());
```

Comment:
We usually dont add console.log statements. We might use them for debugging purposes, but you can also consider this plugin: http://extension.remotedev.io/

It is also mentioned in the troubleshooting section of our documentation: https://awslabs.github.io/diagram-maker/usage/troubleshooting.html#redux-devtools-extension.